fe/fe-core/src/main/java/org/apache/doris/analysis/Analyzer.java

```diff
@@ -680,7 +680,7 @@ public TableRef resolveTableRef(TableRef tableRef) throws AnalysisException {
         // which may not be the same as the user's reference to the table name, causing the table name not to be found
         // in registerColumnRef(). So here the tblName is constructed using tableName.getTbl()
         // instead of table.getName().
-        TableName tblName = new TableName(tableName.getDb(), tableName.getTbl());
+        TableName tblName = new TableName(tableName.getCtl(), tableName.getDb(), tableName.getTbl());
```

Comment:
I think we should disable the constructor of `TableName` with 2 params.
That is very error-prone